wpengine/faustjs
# [MERL-441] WordPress RSS feeds

## Description

This change prevents WordPress feeds from redirecting to the front-end application. This resolves https://github.com/wpengine/faustjs/issues/969 - props to [@julienkoss](https://github.com/julienkos) for the feedback!

## Testing

1. Pull down changes from the `MERL-441-rss` branch.
2. Go to `/feed` on your WordPress site.
3. Notice that the RSS feed for your WordPress site is accessible.
